packages/zosuss/src/Shell.ts

```diff
@@ -94,8 +94,14 @@ export class Shell {
                                 dataToPrint = "";
                                 isUserCommand = false;
                             } else if (isUserCommand) {
-                                // print out the user command result
-                                stdoutHandler(dataToPrint);
+                                // don't print out command (starts with `$`) just print result
+                                if (!dataToPrint.match(/\$/)){
+                                    if (dataToPrint.startsWith("\r\n")){
```

Comment:
The issue with the original command being included in the response does not occur on all systems.
Not sure why but I haven't been able to reproduce it.
When I test this PR locally, it breaks the SSH command by removing the first line of command output.

We should only strip off the first line of data in situations where we are confident it can be discarded without losing data.
Perhaps we can isolate the types of systems where the issue occurs? For example maybe it is specific to Windows.
Or we could make this logic more specific by checking whether `dataToPrint` begins with the command that the user entered.

Reply:
😅

Reply:
`zowe zos-ssh issue cmd "pwd"` works fine now 🙂 
But if I run a command that outputs multiple lines like `pwd && echo hello`, the 2nd line of output is lost.

I think some additional logic may be needed so that we check for the command to be present only for the first line of output, rather than for all lines?

Reply:
Should work now, try again. I removed handling new line characters since we really can't anticipate how users will interact with this command